hack/fuse-demo/wrap_datamon.sh

```diff
@@ -412,7 +412,7 @@ while [[ -z $MOUNT_COORD_DONE ]]; do
             # mount point already found
             continue
         fi
-        if echo "$mount_data" | grep -q "^$mount_point fuse$"; then
+        if echo "$mount_data" | grep -q "^$mount_point fuse.datamon$"; then
```

Comment:
what happened here?

Reply:
>   core: set fuse moumt subtype to "datamon" and "datamon-mutable", so it appears as such in the output of the "mount" UNIX command

Experimenting things on my own box (which has a lot of other fuse mounts) gave me the idea of giving a proper signature to _our_ mounts. I then realized you had a strict check there, so updated this.

Reply:
i've since found the change to `core/fs.go`.

and i'm pretty sure the ci tests cover the pipeline tool's expectations.

in any case, i'll know to keep an eye out for this change in case it affects the Argo-ization anywhere else